css/style.css

```diff
@@ -122,15 +208,15 @@ body{
         margin-top: 150px;
     }
 
+    .combine_emelent_with_circle{
```

Comment:
Interesting things with class naming) pls check again BEM approach and try to implement it)